app/views/docs/databases.phtml

```diff
@@ -319,25 +312,49 @@ func main() async throws{
             <td>Not equal to.</td>
         </tr>
         <tr>
-            <td>lesser</td>
-            <td>Lesser than.</td>
+            <td>lessThan</td>
+            <td>Less than.</td>
         </tr>
         <tr>
-            <td>lesserEqual</td>
+            <td>lessThanEqual</td>
             <td>Less than or equal to.</td>
         </tr>
         <tr>
-            <td>greater</td>
+            <td>greaterThan</td>
             <td>Greater than.</td>
         </tr>
         <tr>
-            <td>greaterEqual</td>
+            <td>greaterThanEqual</td>
             <td>Greater than or equal to.</td>
         </tr>
         <tr>
             <td>search</td>
             <td>Requires a Fulltext Index.</td>
         </tr>
+        <tr>
+            <td>orderDesc</td>
+            <td>Order results in descending order.</td>
+        </tr>
+        <tr>
+            <td>orderAsc</td>
+            <td>Order results in ascending order.</td>
+        </tr>
+        <tr>
+            <td>limit</td>
+            <td>Limits the number of results returned by the query. Used for <a href="/docs/pagination#offset-pagination" id="offset-pagination">pagination</a>.</td>
+        </tr>
+        <tr>
+            <td>offset</td>
+            <td>Offset the results returned by skipping some of the results. Used for <a href="/docs/pagination#offset-pagination" id="offset-pagination">pagination</a>.</td>
+        </tr>
+        <tr>
+            <td>cursorAfter</td>
+            <td>Places the cursor after the specified item ID. Used for <a href="/docs/pagination#cursor-pagination" id="cursor-pagination">pagination</a>.</td>
```

Comment:
```suggestion
            <td>Places the cursor after the specified resource ID. Used for <a href="/docs/pagination#cursor-pagination" id="cursor-pagination">pagination</a>.</td>
```

Not sure if we call it resource. But item also doesn't sound correct. Same for cursorBefore description below this.